examples/serialblaster/serialblaster_main.c

```diff
@@ -84,25 +85,25 @@ int main(int argc, FAR char *argv[])
   fd = open(devpath, O_RDWR);
   if (fd < 0)
     {
-      printf("dev_ttyS2: ERROR Failed to open /dev/ttyS2\n");
+      printf("%s: ERROR Failed to open %s\n", devpath, devpath);
```

Comment:
Why do we need to print `devpath` two times?

Reply:
OK. It is done. Now there is argv[0] and devpath